### DISCUSSION TITLE
MiKTeX doesn't compile

### DISCUSSION THREAD

**Discussion by athan01** [MiKTeX/miktex#689] [Q&A]
Hello everyone,
I installed MiKTeX (Windows 10 x64) and TeXstudio and cannot compile not a even a simple Hello World.
I started first a chat here [https://tex.stackexchange.com/questions/579042/texstudio-miktex-compile-error](url)
because even if I tried all the suggestions found online or what I got feedback from this question it still doesn't work.
There I provided some log files for the error I get. I want to compile with xelatex but none of them work.
I installed all packages, uninstalled and reinstalled some of them who someone suspected they must be corrupted but no.
I really need your help!!!
Thank you

**Comment by edocevoli** [comment]
Often those kind of issues arise from an installation in MiKTeX-Admin mode, i.e., the admin vs user package installation dilemma.

You can try to recreate the format files by running `initexmf --dump` and/or `initexmf --admin --dump`.

**Reply by athan01** [reply]
Hello
At first I chose single user as it is suggested now I installed as admin. I run both but returned sorry, but MiKTeX Configuration Utility did not succeed. I tried many things almost anything but installing to another Drive Path
ReportDate: 2021-01-17 23:50:18
CurrentVersion: 20.12
SetupDate: 2021-01-15 23:37:00
SetupVersion: 20.12
Configuration: Regular
GitInfo: 7017ca6 / 2020-12-13 20:26:45
OS: Windows 10 Pro, 64-bit, build 10240
SharedSetup: yes
LinkTargetDirectory: C:\Program Files\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: 2021-01-16 18:07:04
LastUpdateAdmin: 2021-01-15 23:39:22
LastUpdateDbAdmin: 2021-01-16 18:06:53
SystemAdmin: yes
RootPrivileges: no
AdminMode: no
Root0: C:\Users\Athan\AppData\Roaming\MiKTeX
Root1: C:\Users\Athan\AppData\Local\MiKTeX
Root2: C:\ProgramData\MiKTeX
Root3: C:\Program Files\MiKTeX
UserInstall: C:\Users\Athan\AppData\Roaming\MiKTeX
UserConfig: C:\Users\Athan\AppData\Roaming\MiKTeX
UserData: C:\Users\Athan\AppData\Local\MiKTeX
CommonInstall: C:\Program Files\MiKTeX
CommonConfig: C:\ProgramData\MiKTeX
CommonData: C:\ProgramData\MiKTeX

**Reply by athan01** [reply]
Hello. I installed TeX Live 2020 changed the path to TeXstudio and everything is working. If MiKTeX has more packages why it's half the size of TeX Live? Though at the moment this suits me I'd wish to fix MiKTeX too. I uninstalled evrything again and installed the complete version so as I understand if there is a corrupted file that must be downladed corrupted from the server, right? I cross checked with three CTAN servers the files. I also downloaded protext which includes MiKTeX 2.9 but why downgrade since we need to be updated. And saw this here tex.stackexchange.com/questions/579320/… is it an aswer to my problem too? I think not since he has a problem creating the pdf but mine doesn't compile at all

**Comment by edocevoli** [comment]
If I understand you correctly, `xelatex` does not work for you because the format file `xelatex.fmt` is missing. Have you tried to generate it manually by running

```
initexmf --dump=xelatex
```
?

**Reply by athan01** [reply]
Hi, it returned sorry, but MiKTeX Configuration Utility did not succeed

**Reply by athan01** [reply]
Hello to whom is looking for an answer to a similar question.

The solution in this case is yet to be found. The uninstallation and installation of MiKTeX 21.1 since the 20.10 probably had some corrupted files on the servers is what I tried today with the 21.1 and the new installer 4.2 One thing I have to mention is that every time I launch the console I get a warning to fix the path. I press fix it says has been successfully modified but at next launch the same thing. Everything is working still under TeX Live. I'll install again for all users and see what happens

**Reply by athan01** [reply]
I unistalled again and installed for all users to give me the admin option. That fixed the path to MiKTeX but still the same problem. I inserted manually all the commands and the path to MiKTeX the same problem. I changed them all to TeX Live everything is working. I really would like to use MiKTeX but is it offering more to keep trying or stay with TeX Live.

**Reply by edocevoli** [reply]
Glad to hear that you have a working system now. TeX Live is certainly a good option if you are looking for a hassle free solution.
